gfx_imgui.cpp

```diff
@@ -283,16 +287,7 @@ class GfxImGuiInternal
             gfxCommandSetScissorRect(gfx_); // reset scissor test
         }
 
-        ImGui_ImplWin32_Data* bd = ImGui_ImplWin32_GetBackendData();
-        if(bd != nullptr && bd->hWnd != 0)
-        {
-            POINT pos = {};
-            GetCursorPos(&pos);
-            ImGui_ImplWin32_NewFrame();
-            ScreenToClient(bd->hWnd, &pos);
-            io.MousePos.x = (float)GFX_MIN(GFX_MAX(pos.x, (LONG)0), (LONG)gfxGetBackBufferWidth(gfx_)  - 1);
-            io.MousePos.y = (float)GFX_MIN(GFX_MAX(pos.y, (LONG)0), (LONG)gfxGetBackBufferHeight(gfx_) - 1);
-        }
+        ImGui_ImplWin32_NewFrame();
```

Comment:
This code didnt appear to be needed, imgui appears to handle all this withing `ImGui_ImplWin32_NewFrame`